docs/tails_guide.rst

```diff
@@ -147,37 +147,29 @@ installation by typing these commands into the terminal:
     sudo ./install.sh
 
 Type the administration password that you selected when starting Tails
-and hit enter. The installation process will download additional
-software and then open a text editor with a file called
-*torrc\_additions*.
+and hit enter. This installation script does the following: 
 
-Edit the file, inserting the *HidServAuth* information for your
-SecureDrop instance that you received during the installation process.
-The values can be found in
-``install_files/ansible-base/app-document-aths``. This information
-includes the address to the Document Interface and your personal
-authentication string, as seen in the example below:
+* Downloads additional software
+* Installs a program that automatically and persistently configures Tor to access the
```

Comment:
@garrettr I think of the NM hooks as contingent on the network connection in the session, rather than part of the "boot" process.